src/components/UIComponent/SideBarItem.tsx

```diff
@@ -42,20 +42,18 @@ const ImgLogo = styled.img.attrs({
   margin-left: auto;
   margin-right: auto;
 `;
-type OverlayProps = {
-  userId: string;
-  experienceData: ExperienceData;
-};
-function SideBarItem({userId, experienceData}: OverlayProps): JSX.Element {
+
+function SideBarItem(): JSX.Element {
   const { toggleApparatusInfo, toggleSkyBoxInfo } = useContext(SideBarContext);
-  const { actionList } = useActionList(experienceData);
+  const { experienceData, userId , setExperienceData }: globalContextTypes = useContext(GlobalContext);
+  const { actionList } = useActionList(experienceData,setExperienceData);
```

Comment:
this shouldn't use the useActionList hook but the context

Reply:
seems to have been resolved ,  could you pull and try again?